test/controllers/ViewTransfersControllerSpec.scala

```diff
@@ -89,15 +92,16 @@ class ViewTransfersControllerSpec extends FrontEndTestHelper {
   }
 
   "ViewTransfersController" should {
-    "render the view transfers page with a list of all user's consignments" in {
+    "render the view transfers page with a list of all user's consignments and should not display pagination when total number of pages is 1" in {
       setConsignmentTypeResponse(wiremockServer, standardType)
       val consignments = setConsignmentViewTransfersResponse(wiremockServer, standardType, statuses = List())
 
       val graphQLConfiguration = new GraphQLConfiguration(app.configuration)
       val consignmentService = new ConsignmentService(graphQLConfiguration)
-      val controller = new ViewTransfersController(consignmentService, getValidStandardUserKeycloakConfiguration, getAuthorisedSecurityComponents)
+      val applicationConfig = new ApplicationConfig(app.configuration)
+      val controller = new ViewTransfersController(consignmentService, applicationConfig, getValidStandardUserKeycloakConfiguration, getAuthorisedSecurityComponents)
       val response = controller
-        .viewConsignments()
+        .viewConsignments(1)
```

Comment:
The 1 is redundant since, you've already set the controller method with a default of 1